FLATApp/FLATApp/IDsearchView.swift

```diff
@@ -87,6 +91,17 @@ struct IDsearchView: View { //友達追加画面
             }
         }.resume()
     }
+    
+    private func validateName() {//バリデーションチェック
+        if self.user.name.count == 0 {
+            self.errorMessage = "名前を入力してください。"
+            self.showError = true
+        }
+        if   self.user.name.count < 0 || self.user.name.count >= 10 {
```

Comment:
```suggestion
        if self.user.name.isEmpty || self.user.name.count > 10 {
```
とかで動くかな？nameがString型ならnilが許容されていないのでisEmptyで判定したほうが見やすそう
それと、.countが0未満を吐く可能性が無いと思ったので(もしあったらごめん)